client/templates/pairs.js

```diff
@@ -48,6 +36,10 @@ Template.pairs.events({
   "click .pair-it": function (event) {
     generatePairs();
     return false;
+  },
+  "click .full-it": function (event) {
+    render('fullscreen');
```

Comment:
Since this event is for navigation, it would probably be simpler if we removed this JavaScript code and used an `<a>` tag instead.